snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -0,0 +1,103 @@
+# Profile for F5 BIG-IP devices
+#
+# You need the MIB compiled to get the data. For example, run:
+# $ /opt/datadog-agent/embedded/bin/python /opt/datadog-agent/embedded/bin/mibdump.py  --destination-directory=/opt/datadog-agent/embedded/lib/python2.7/site-packages/pysnmp_mibs F5-BIGIP-SYSTEM-MIB
+#
+# Then you can use it this way in your SNMP configuration:
+# init_config:
+#   profiles:
+#     f5-big-ip:
+#       definition_file: profiles/f5-big-ip.yaml
```

Comment:
Good job for keeping the yaml extension 👍 
I think that it would be easier for users if they could just write `definition_file: f5-big-ip.yaml` and the integration would go looking into that profiles folder automatically.

Reply:
I feel that would be make it a bit too magical. Right now it "just" tries to handle relative paths, but if you want to use absolute path it'll be more explicit?

Reply:
🤔 Mmh I see, but I think this is already a bit magical that relative paths starts from the config file and not the agent binary for example. That could be a possibility to explicitly state  and document that the snmp integration will look into a the "profiles" directory, that would remove some of the magic. Also ith relative paths starting from the `conf.yaml` file, some users might be tempted to add a yaml profile in that directory (which would break the agent).

There are pros and cons in both, I don't have a strong opinion here, just raising some potential issues :) Great PR!

Reply:
OK that's fair.

Reply:
Seems @FlorianVeaux mean to change `filename = os.path.join(confd, 'snmp.d', filename)` => `filename = os.path.join(confd, 'snmp.d', 'profiles', filename)`: 

https://github.com/DataDog/integrations-core/blob/171478e0feb853332dd7815eb4888e92c9f1a7e1/snmp/datadog_checks/snmp/snmp.py#L79